test/clojure/nrepl/core_test.clj

```diff
@@ -70,7 +72,8 @@
 
 (defmacro def-repl-test
   [name & body]
-  `(deftest ~(with-meta name (merge {:private true} (meta name)))
+  ;; TODO: decide what to do with the private tags before merging
+  `(deftest ~(with-meta name (merge #_{:private true} (meta name)))
```

Comment:
Why do we make the repl tests private? It stops one from using test selectors like this from working:

```
lein with-profile +1.10,+test test :only nrepl.core-test/transports-fail-on-disconnects
```

I'd like to remove the private bit if possible. This is also the one line that's generating eastwood warnings.

Reply:
No idea. Likely some oversight. Feel free to remove this.

Reply:
👍